### PR TITLE
ANC PostPartum - Breastfeeding

### PR DESCRIPTION
**Pull request by anvmn**
#362 
#363
#364

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-08-09 22-31

**Comment by anvmn** [comment]
@nedSaf please review

File: client/src/elm/Pages/Prenatal/Utils.elm [hunk 1]
```diff
@@ -2393,6 +2468,11 @@ resolveVitaminADosageAndIcon currentDate person =
     Just ( "200,000 IU", "icon-pills" )
 
 
+resolveParacetamolDosageAndIcon : NominalDate -> Person -> Maybe ( String, String )
+resolveParacetamolDosageAndIcon currentDate person =
```
Comment (nedSaf) [inline client/src/elm/Pages/Prenatal/Utils.elm:2472]: Those variables don't seem to be used here, do you want to keep them?

**Review by nedSaf — APPROVED** [review]
Looks good, minor comment.

File: client/src/elm/Pages/Prenatal/Utils.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Pages/Prenatal/Utils.elm:2472]: @nedSaf 

It's in use:

Selection_1030

Comment (nedSaf) [reply client/src/elm/Pages/Prenatal/Utils.elm:2472]: Not the function, the variables of this function: `currentDate` & `person`

Comment (anvmn) [reply client/src/elm/Pages/Prenatal/Utils.elm:2472]: @nedSaf Ah, I see.
We have many places where passed variables are not in use.
No need to change - we pay nothing for passing them, and may need them in future.

Comment (nedSaf) [reply client/src/elm/Pages/Prenatal/Utils.elm:2472]: 👍 Okay, sounds good.

**Pull request merged**